a10_neutron_lbaas/v1/handler_hm.py

```diff
@@ -13,36 +13,18 @@
 #    under the License.
 
 import acos_client.errors as acos_errors
-import handler_base
+
+import a10_neutron_lbaas.a10_openstack_map as a10_os
+import handler_base_v1
 import v1_context as a10
 
 
-class HealthMonitorHandler(handler_base.HandlerBase):
+class HealthMonitorHandler(handler_base_v1.HandlerBaseV1):
 
-    def _hm_name(self, hm):
+    def _name(self, hm):
         return hm['id'][0:28]
```

Comment:
Perhaps store the slice values in constants so we know why you're getting 29 chars?